packages/indexer-client/src/types/clientTypes.ts

```diff
@@ -473,3 +474,26 @@ export type GetIndexerClaimArbMerkleProofsParams =
 
 export type GetIndexerClaimArbMerkleProofsResponse =
   GetIndexerClaimVrtxMerkleProofsResponse;
+
+export type GetIndexerMakerStatisticsParams =
```

Comment:
Unfortunately a bit annoying but perhaps we want to rename `product_id` to be camelCase here